### ISSUE TITLE
Error when following the Sample Data Page 

### ISSUE THREAD

**Issue by Solicus** [influxdata/docs.influxdata.com#2336]
I am fairly new to InfluxDB and I am trying to load sample data into a database. 
I am using the latest version of InluxDB on Windows. 

Here is the page I am following: https://docs.influxdata.com/influxdb/v1.7/query_language/data_download/

After loading the data in the terminal with:
curl https://s3.amazonaws.com/noaa.water-database/NOAA_data.txt -o NOAA_data.txt

When I am trying to write data into influx via the influxdb shell with:
influx -import -path=NOAA_data.txt -precision=s -database=NOAA_water_database

I am getting this error: 
ERR: error parsing query: found influx, expected SELECT, DELETE, SHOW, CREATE, DROP, EXPLAIN, GRANT, REVOKE, ALTER, SET, KILL at line 1, char 1

Somebody got a solution for this?

**Comment by stevebang** [comment]
OK, there are a few unresolved questions here, but I tested the steps on my Mac and learned a few things that might be relevant... @Solicus might have tried running the `influx -import ...` command from within the shell ... that's the only way I could reproduce that error message. Running the command without the `-database` argument still generates that error, if run within the shell.

On my Mac, I tested using the `influx -import ...` from the docs, with and without the `-database` argument, and the database gets created and loaded without errors. The `NOAA_data.txt` file includes a DDL statement that creates the database. So, I can remove that argument in the command and it sounds like it will work OK on Macs and Windows then. I'll modify the steps so they're a bit clearer too.
